guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,300 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
 
-### Earning bitcoin
+Below we explore a standard funding approach that does not make use of any Lightning services. Lightning services are services that can be offered by your application that avoid users having to manually manage and configure their wallet. We explore the user flows around funding with Lightning services in our using Lightning services page. 
 
-Earning bitcoin through exchange of goods and services is a great option. Some individuals and businesses have already started accepting and even paying employees in bitcoin.
+The below flow expands on what we covered in the previous page covering the users [first use]({{'/guide/onboarding/first-use/' | relative_url}}).
 
-> Fun Fact 🏈: Former Seattle Seahawk Russel Okung puts half of his salary in Bitcoin and is now considered one of the highest paid players in the league.
->
-> <cite>As reported by <a href="https://www.nbcsports.com/northwest/seahawks/former-seattle-seahawk-russell-okung-puts-half-salary-bitcoin-considered-highest">NBC Sports</a></cite>
+## Do your users own bitcoin?
 
-{% include fact/pros.html %}
- - Privacy
- - No exchange fees
- - Simplifies onboarding
- {% include fact/close.html %}
+The first step in having users fund your wallet is finding out if they currently own bitcoin or not. This will determine what kind of information to present to your users next as well as guide them towards how they will fund their wallet. If the user wants to fund their wallet at a later time they should have the option to skip this process and jump straight into the wallet.
 
-{% include fact/cons.html %}
- - Managing taxes
- - Difficult to find businesses/individuals willing to pay in bitcoin
- - Limited options for recurring payments
- {% include fact/close.html %}
+If your user does not have any bitcoin, your application should guide them in obtaining some. This could be done by recommending users find a local exchange to buy bitcoin from and to come back once they have some before proceeding. Alternatively, you could offer the option to buy directly within your application to speed up this process.
 
-### Buying bitcoin
+<div class="image-slide-gallery">
 
-<div class="center" markdown="1">
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/own-bitcoin@2x.png"
+   layout = "shadow"
+   caption = "Ask the user if they own any bitcoin."
+   alt-text = "Screen asking the user if they own any bitcoin."
+   width = 250
+   height = 541
+%}
 
-{% include image.html
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/where-to-buy.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/where-to-buy@2x.png"
+   layout = "shadow"
+   caption = "If not, guide them on where to get some."
+   alt-text = "Screen guiding user where to buy bitcoin."
+   width = 250
+   height = 541
+%}
```

Comment:
It's too hard to recommend places unless the wallet is only located in one location. But agree we could have a better solution here. 

Personally though I think Googling would be the number one way people do find exchanges to buy bitcoin though despite the scams (I don't think it's that rampant of an issue).

Reply:
This is a situation where a crowdsourced community user guide would be helpful. Like a design guide, but it would focus on how-to for users. Wallets could link out to it for various things like backup tutorials, how-to-buy bitcoin, etc. Doesn't make much sense to me that all wallets need to recreate this stuff. Not sure how much of this already exists in one form or another.

Reply:
I think that a wallet optionally integrating a way for users to buy btc can be a good revenue stream for them as well as a smooth UX.

Reply:
I agree with @pavlenex.

- If the user is inexperienced enough to where they need to be told to find bitcoin via a Google search, then they are inexperienced enough to need to be guarded from accidentally stumbling into a scam or accidentally acquiring Bitcoin Cash or Doge.
- If the user is experienced enough to be trusted finding bitcoin via a Google search, then they are so experienced they don't need to be told where to find bitcoin.

You do raise a good point @Bosch-0 about exchanges not working in every jurisdiction. It would be a very US-centric worldview to simply declare "just use XYZ Exchange". Nonetheless, I think the risk of accidentally sending the user down a Google rabbit hole is worse.

One potential solution: wallet has an (entirely optional) search function where the user can seek out an exchange. "Select your country" to find a recommended exchange.

Reply:
What about just making this generic and mentioning it's up to the application to recommend where to send users to buy?